front/src/pages/IndexPage.vue

```diff
@@ -1,19 +1,24 @@
 <template>
+  <div class="q-pa-md" style="display: flex; justify-content: space-between">
```

Comment:
```suggestion
  <div class="q-pa-md" style="display: flex; justify-content: space-between; max-width: 1480px">
```

Se você colocar no pai, não precisa replicar para os filhos. Ai pode tirar as que estão dentro dessa div. 

Acho legal separar o css do html também, da para você criar as classes e aqui só jogar o nome delas